internal/policy/numericall_test.go

```diff
@@ -0,0 +1,147 @@
+/*
```

Comment:
The name of this file does not match the name of the file it tests (`numerical` vs `numericall`).

Reply:
And here I was thinking it was all perfect and fine. Thanks!